src/services/MarkdownRenderer.ts

```diff
@@ -58,9 +59,11 @@ export class MarkdownRenderer {
 
   private headingEnhanceRenderer: marked.Renderer;
   private originalHeadingRule: typeof marked.Renderer.prototype.heading;
+  parser: marked.Parser;
 
   constructor(public options?: RedocNormalizedOptions) {
     this.headingEnhanceRenderer = new marked.Renderer();
+    this.parser = new marked.Parser();
```

Comment:
It seems it is not used anywhere... ?

Reply:
it is used by `marked.Renderer()` under the hood. Without this initialization, `marked` will not have context of the parser

Reply:
Add a comment please. Otherwise someone will remove it 😄